File: java/lib/src/test/java/org/xmpt/proto/message/contents/ProtoTest.java
package org.xmpt.proto.message.contents;

import org.junit.Test;
import org.xmtp.proto.message.contents.*;
import static org.junit.Assert.*;

public class ProtoTest {
    @Test public void versionMethodReturnsTrue() {
        assertEqual(Content.Compression.COMPRESSION_DEFLATE.getNumber(), 0);
    }
}

File: kotlin/lib/src/test/kotlin/org/xmtp/proto/message/ProtoTest.kt
package org.xmtp.proto.message

import org.xmtp.proto.message.api.v1.MessageApiOuterClass.SortDirection
import kotlin.test.Test
import kotlin.test.assertEquals

class ProtoTest {
    @Test fun testTypesAreAvailable() {
        assertEquals(1, SortDirection.SORT_DIRECTION_ASCENDING_VALUE)
    }
}